crates/solidity/testing/snapshots/bindings_output/contracts/storage_layout_constant/input.sol

```diff
@@ -2,3 +2,7 @@ uint256 constant C1 = 0;
 
 contract X1 layout at C1 {
```

Comment:
Suggestion: splitting such tests into two (positive that should bind, and negative that shouldn't bind).
This will make it easier to review/detect bugs in the future.